internal/controller/ownerref.go

```diff
@@ -73,3 +74,65 @@ func findFinalOwnerRef(ctx context.Context, client crclient.Client, ns string, o
 	Log.V(1).Info("Pods owner found", "Kind", ownerRef.Kind, "Name", ownerRef.Name, "Namespace", ns)
 	return ownerRef, nil
 }
+
+//FindControllerOwner returns OwnerReferent that owns controller process
+//it starts searching from the current pod and then resolves owners recursively
+//until object without owner is not found
```

Comment:
In addition to @sleshchenko's comment, I prefer comments formatted with a space:
```suggestion
// FindControllerOwner returns OwnerReferent that owns controller process
// it starts searching from the current pod and then resolves owners recursively
// until object without owner is not found
```